src/nanovg_gl.h

```diff
@@ -1071,9 +1071,11 @@ static void glnvg__renderFlush(void* uptr)
 
 	if (gl->ncalls > 0) {
 
-		// Setup require GL state.
+		// Save origin GL Shader
+		GLint old_program;
```

Comment:
This is good change (even if the getters are often slow). Could you change `old_program` to `oldProgram` to follow the general style (I might have alternatively written `oldprog` too).